test/schema/collection.js

```diff
@@ -0,0 +1,72 @@
+import { resolve } from "path"
+import { readFileSync } from "fs"
+
+const gravityData = {
+  id: "saved-artwork",
+  name: "Saved Artwork",
+  default: true,
+  description: "",
+  image_url: null,
+  image_versions: null,
+  private: false,
+}
+
+describe("Collections", () => {
+  describe("Handles getting collection metadata", () => {
+    const Collection = schema.__get__("Collection")
+    let gravity = null
+    beforeEach(() => {
+      gravity = sinon.stub()
+      gravity.with = sinon.stub().returns(gravity)
+
+      Collection.__Rewire__("gravity", gravity)
+    })
+
+    afterEach(() => {
+      Collection.__ResetDependency__("gravity")
+    })
+
+    it.skip("returns collection metadata", () => {
```

Comment:
Is this a pended test? Also, since `user_id` is required for the collection endpoint, and presumably you should have one in this context, maybe the specs should just have `'orta'` or something other than null?